src/test/java/net/sf/jabref/logic/formatter/OrdinalsToSuperscriptFormatterTest.java

```diff
@@ -2,29 +2,15 @@
 
 
 import net.sf.jabref.logic.formatter.bibtexfields.OrdinalsToSuperscriptFormatter;
-import org.junit.After;
 import org.junit.Assert;
-import org.junit.Before;
 import org.junit.Test;
 
+/**
+ * Tests in addition to the general tests from {@link net.sf.jabref.logic.formatter.FormatterTest}
+ */
 public class OrdinalsToSuperscriptFormatterTest {
-    private OrdinalsToSuperscriptFormatter formatter;
 
-    @Before
-    public void setUp() {
```

Comment:
Using the setup method is the preferred way to init objects under test (since then every test method gets a new object and possible modifications from a previous test are erased). This is probably not a big deal for the formatters but I wouldn't break the pattern here.